i18n/en-US.properties

```diff
@@ -10,6 +10,8 @@ ba.annotationsPost = Post
 ba.annotationsSave = Save
 # Aria label description for reply popup
 ba.popup.reply.comment = Comment
+# Aria label description for annotation comment field
+ba.popup.reply.field = Type a comment
```

Comment:
Was this message given by the accessibility or messaging team? If not do we need to add something more specific like "Type an annotation comment"?

Reply:
Actually it was not given, I came up with it. I will change it to "Type an annotation comment" to be more specific

Reply:
Hm I see, I think it should be okay then if that's the case